src/main/java/io/jenkins/plugins/checks/github/status/GitHubStatusChecksConfigurations.java

```diff
@@ -66,5 +70,15 @@ public boolean isSuppressLogs() {
     public boolean isSkipProgressUpdates() {
         return false;
     }
+
+    @Override
+    public boolean isPublishConfluentIncPR() {
+        return false;
+    }
+
+    @Override
+    public boolean isPublishNonConfluentIncPR() {
+        return false;
+    }
```

Comment:
just to confirm, does this default configuration prevent the running & publishing of results on the non-public CI system?